python/paddle_fl/mpc/layers/basic.py

```diff
@@ -129,3 +130,21 @@ def elementwise_mul(x, y, axis=-1, act=None, name=None):
     Examples: todo
     """
     return _elementwise_op(MpcLayerHelper('elementwise_mul', **locals()))
+
+
+def share(input, role=0, act=None, name=None):
```

Comment:
the name 'role' here is confused to user.

Reply:
fixed